src/zenml/zen_stores/secrets_stores/rest_secrets_store.py

```diff
@@ -148,7 +148,7 @@ def _initialize(self) -> None:
     # Secrets
     # ------
 
-    @track(AnalyticsEvent.CREATED_SECRET)
+    @track(AnalyticsEvent.CREATED_SECRET, v2=True)
```

Comment:
@stefannica Should this be on the rest secrets store? I think that one is only client side (similar to the rest zen store) and should not track anything?

Reply:
Quite true, the rest secrets store is the only store where we don't track any events.

Reply:
That depends. Don't we track any client-side analytics through the V2 analytics ? If that's the case, then you're right, no v2 events should be generated in the REST zen store.

Reply:
I think we still track some events client side which can't be tracked on the server (e.g. `zenml go`), but events related to the ZenStore/SecretsStore should be tracked server side. I'll remove it on the rest implementation then.